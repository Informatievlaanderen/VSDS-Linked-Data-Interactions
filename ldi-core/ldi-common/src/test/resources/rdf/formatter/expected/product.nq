<http://example.org/cars/for-sale#tesla> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/goodrelations/v1#Offering> .
<http://example.org/cars/for-sale#tesla> <http://purl.org/goodrelations/v1#acceptedPaymentMethods> <http://purl.org/goodrelations/v1#Cash> .
<http://example.org/cars/for-sale#tesla> <http://purl.org/goodrelations/v1#description> "Need to sell fast and furiously"^^<http://www.w3.org/2001/XMLSchema#string> .
<http://example.org/cars/for-sale#tesla> <http://purl.org/goodrelations/v1#hasBusinessFunction> <http://purl.org/goodrelations/v1#Sell> .
<http://example.org/cars/for-sale#tesla> <http://purl.org/goodrelations/v1#hasPriceSpecification> _:genid1 .
<http://example.org/cars/for-sale#tesla> <http://purl.org/goodrelations/v1#includes> _:genid2 .
<http://example.org/cars/for-sale#tesla> <http://purl.org/goodrelations/v1#name> "Used Tesla Roadster"^^<http://www.w3.org/2001/XMLSchema#string> .
_:genid1 <http://purl.org/goodrelations/v1#hasCurrency> "USD"^^<http://www.w3.org/2001/XMLSchema#string> .
_:genid1 <http://purl.org/goodrelations/v1#hasCurrencyValue> "85000"^^<http://www.w3.org/2001/XMLSchema#float> .
_:genid2 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/goodrelations/v1#Individual> .
_:genid2 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.productontology.org/id/Vehicle> .
_:genid2 <http://purl.org/goodrelations/v1#name> "Tesla Roadster"^^<http://www.w3.org/2001/XMLSchema#string> .
_:genid2 <http://xmlns.com/foaf/0.1/page> <http://www.teslamotors.com/roadster> .