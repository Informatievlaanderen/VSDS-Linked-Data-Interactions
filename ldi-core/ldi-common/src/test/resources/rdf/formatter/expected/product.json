{
  "@context" : {
      "gr" : "http://purl.org/goodrelations/v1#" ,
      "pto" : "http://www.productontology.org/id/" ,
      "foaf" : "http://xmlns.com/foaf/0.1/" ,
      "xsd" : "http://www.w3.org/2001/XMLSchema#" ,
      "foaf:page" : {"@type" : "@id" } ,
      "gr:acceptedPaymentMethods" : {"@type" : "@id" } ,
      "gr:hasBusinessFunction" : {"@type" : "@id" } ,
      "gr:hasCurrencyValue" : {"@type" : "xsd:float" }
    } ,
  "@id" : "http://example.org/cars/for-sale#tesla" ,
  "@type" : "gr:Offering" ,
  "gr:acceptedPaymentMethods" : "gr:Cash" ,
  "gr:description" : "Need to sell fast and furiously" ,
  "gr:hasBusinessFunction" : "gr:Sell" ,
  "gr:hasPriceSpecification" : {
      "@id" : "_:b1" ,
      "gr:hasCurrency" : "USD" ,
      "gr:hasCurrencyValue" : "85000"
    } ,
  "gr:includes" : {
      "@id" : "_:b0" ,
      "@type" : [
          "pto:Vehicle" ,
          "gr:Individual"
        ] ,
      "gr:name" : "Tesla Roadster" ,
      "foaf:page" : "http://www.teslamotors.com/roadster"
    } ,
  "gr:name" : "Used Tesla Roadster"
}